torch/csrc/inductor/aot_runtime/interface.h

```diff
@@ -52,9 +53,13 @@ AOTInductorError AOTInductorModelContainerDelete(
 // Runs the inference.
 AOTInductorError AOTInductorModelContainerRun(
     AOTInductorModelContainerHandle container_handle,
-    AOTInductorTensorHandle input_handles,
+    // array of raw AtenTensorHandle for input tensors, and will be stolen by
+    // RAIIAtenTensorHandle
+    AtenTensorHandle* input_handles,
     size_t num_inputs,
-    AOTInductorTensorHandle output_handles,
+    // array of raw AtenTensorHandle for output tensors, and will be stolen by
+    // RAIIAtenTensorHandle
+    AtenTensorHandle* output_handles,
```

Comment:
BTW, it's kind of weird to steal the output handles, since for these we definitely know they want to stay live beyond the extent of the function.